IaC/pmm.cd/init.groovy.d/cloud.groovy

```diff
@@ -317,6 +325,7 @@ devMap['docker']            = '/dev/xvda=:8:true:gp2,/dev/xvdd=:80:true:gp2'
 labelMap = [:]
 labelMap['min-centos-7-x64']  = 'min-centos-7-x64'
 labelMap['min-centos-8-x64']  = 'min-centos-8-x64'
+labelMap['min-centos-9-x64']  = 'min-centos-9-x64'
```

Comment:
@nikita-b it's quite confusing to have a "centos-9" label here, and completely different system under the hood

Reply:
These labels are not only for me but for QA for example.

Different labels will be less obvious for other teams. I can change it to `min-rhel-7-x64` and add notice to docs.

Reply:
Well QA should also update their pipelines